package.json

```diff
@@ -19,7 +19,7 @@
     "node": ">=0.8"
   },
   "dependencies": {
-    "backbone": "1.1.2",
+    "backbone": "1.2.3",
```

Comment:
We've always locked versions with Backbone since it doesn't follow semantic versioning. We can bump it outside of this PR (before tagging a new version).